app/components/tutorial-dialog.tsx

```diff
@@ -20,31 +20,31 @@ const steps = [
     title: "Massive Multiplayer Mines",
     description:
       "Work together in real time to clear the grid and avoid the mines!",
-    icon: <UsersIcon className="w-12 h-12 mb-4 text-primary" />,
```

Comment:
you can actually remove `text-primary` since it was used when we had CSS variables